### PR TITLE
Unit tests are reorganized

### PR DESCRIPTION
**Pull request by UkcaGreen**
Unit tests are reorganized and multiple instances are merged into one file.

### CODE REVIEW

**Comment by hazerb** [comment]
Reorganized tests seems okay to me.

**Comment by DogukanKalkan** [comment]
unit_test_crypto_currency.py works okay.
unit_test_scholarly.py is filled with irrelevant functions.
The function isCoronavirusJson in unit_test_covid_19 seems to be not working.

**Comment by EnesToptas** [comment]
> unit_test_crypto_currency.py works okay.
> unit_test_scholarly.py is filled with irrelevant functions.
> The function isCoronavirusJson in unit_test_covid_19 seems to be not working.

Can you state why you think they are irrelevant? @DogukanKalkan

**Comment by DogukanKalkan** [comment]
> 
> 
> > unit_test_crypto_currency.py works okay.
> > unit_test_scholarly.py is filled with irrelevant functions.
> > The function isCoronavirusJson in unit_test_covid_19 seems to be not working.
> 
> Can you state why you think they are irrelevant? @DogukanKalkan

There must be a confusion between unit_test_scholarly.py and scholarly_unit_test. 

One of them must be selected and filled with test code

**Comment by EnesToptas** [comment]
> > > unit_test_crypto_currency.py works okay.
> > > unit_test_scholarly.py is filled with irrelevant functions.
> > > The function isCoronavirusJson in unit_test_covid_19 seems to be not working.
> > 
> > 
> > Can you state why you think they are irrelevant? @DogukanKalkan
> 
> There must be a confusion between unit_test_scholarly.py and scholarly_unit_test.
> 
> One of them must be selected and filled with test code

Oh, yes, lol. I believe fixed them now. Could you review the latest version.

**Comment by DogukanKalkan** [comment]
The latest version still seems to be not working.

The function isCoronavirusJson in unit_test_covid_19 is the root of the problem.

File: practice-app/unit_test_covid_19.py [hunk 1]
```diff
@@ -37,13 +37,42 @@ def isPlotDataFetchCorrect():
     if not type(json[0]["Cases"])==int:
         return False
     return True
+
+
+def isCoronavirusJson():
```
Comment (DogukanKalkan) [inline practice-app/unit_test_covid_19.py:42]: This function does not work properly. Either it must be deleted or fixed.

**Comment by hazerb** [comment]
Some forgotten things corrected in the code

**Comment by DogukanKalkan** [comment]
All the codes are now in the correct files, and the tests resul in success.

**Pull request merged**